src/PidManager.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-swoole for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-swoole/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Swoole;
+
+use function file_put_contents;
+use Psr\Log\LoggerInterface;
+
+class PidManager
+{
+
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    protected $logger;
+
+    protected $pidFile = '';
+
+    /**
+     * PidManager constructor.
+     *
+     * @param \Psr\Log\LoggerInterface $logger
+     */
+    public function __construct($pidFile, LoggerInterface $logger = null)
```

Comment:
Remove the annotation.

Add a scalar type for `$pidFile` (I'm assuming `string`).